go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -170,12 +193,25 @@ func createPerNodePhysicalVIPs(isIPv6 bool, protocol v1.Protocol, sourcePort int
 	}
 
 	for _, gatewayRouter := range gatewayRouters {
+
+		// Get normal per GWR load-balancers
 		gatewayLB, err := gateway.GetGatewayLoadBalancer(gatewayRouter, protocol)
 		if err != nil {
 			klog.Errorf("Gateway router %s does not have load balancer (%v)",
 				gatewayRouter, err)
 			continue
 		}
+
+		if nodeLocal {
+			// Get node-local per GWR load balancers only
+			gatewayLB, err = gateway.GetGatewayLoadBalancer(fmt.Sprintf("%s_local", gatewayRouter), protocol)
```

Comment:
You should use `GWRouterLocalLBPostfix` here

Reply:
Still using _local

Reply:
need to fix this